AppData/PackageSources.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file shouldn't be here, perhaps you have both your repo and the avalonstudio files in the same place?